special-pages/pages/history/styles/history-theme.css

```diff
@@ -67,4 +67,63 @@ body {
     --history-text-normal: var(--color-white-at-84);
     --history-text-invert: var(--color-black-at-84);
     --history-text-muted: var(--color-white-at-60);
-}
\ No newline at end of file
+}
+
+/**
+ * The scrollbar is a custom scroller that is used in the main
+ * content area and the sidebar nav.
+*/
+.customScroller {
```

Comment:
what’s the reasoning for pulling this up to the theme file? previously we used these *theme files to contain just CSS custom properties I think, with a plan to (hopefully) unify/remove them later.

And then anything ‘global’ in nature (in related to app in question) can live in the App.module.css?

Reply:
I lifted it to the theme file because it is now used in 2 locations on the app, and was following the common 1:1 pairing of js:css files. I can move it back to `App.module.css` then import it to `Sidebar.js` as a second style sheet.